app/src/main/java/com/pvp/app/service/NotificationServiceImpl.kt

```diff
@@ -193,4 +215,36 @@ class NotificationServiceImpl @Inject constructor(
             it.cancel()
         }
     }
+
+    override suspend fun getNotificationForTask(
```

Comment:
Could be single line (same in whole interface and other methods that contain only 1 parameter, to be honest).